tips-and-tricks.md

```diff
@@ -35,12 +35,20 @@ Your can use `srun` to launch an interactive job.
 For example, if you want to use a node with 4 GPUs during 1
 hour, you can type:
 ```
-srun --ntasks=1 --gres=gpu:4 --time=01:00:00 --pty bash -i
+srun --ntasks=1 --cpus-per-task=40 --gres=gpu:4 --time=01:00:00 --qos=qos_gpu-dev --pty bash -i
 ```
 
 Now, you have a brand new shell on a compute node where you can run your scripts interactively
 during 1h.
 
+### How to connect to the node of a launched GPU job
+
+To connect to the node of a launched GPU job you need to make sure that you don't ask for any GPU.
```

Comment:
If you could add something about the use case here, that'd be great ! IIRC that was to monitor the host RAM right ?

Reply:
Also I did not tell you I think but I think you can do some monitoring with Dask, but I would need to find some time to remember how to do it.

I would think the simplest thing to do is to use `client.run` this way you can get nvidia-smi info back to the login node every few seconds or so while you are waiting for the futures objects

Reply:
I seem to remember there were more clever way to do it but I am not sure.

Also super nice about Dask : the diagnostics dashboard: https://docs.dask.org/en/latest/diagnostics-distributed.html. You can play with it locally (`LocalCluster`) or directly on binder: https://mybinder.org/v2/gh/dask/dask-examples/master?urlpath=lab/tree/futures.ipynb

Reply:
it was indeed about RAM manual monitoring. I added it !

I will need to look into dask monitoring at some point, it was one of the points that made me want to try it.

Reply:
Great let me know when you get the chance or you have questions !